packages/drivers/routerlicious-driver/src/test/errorUtils.spec.ts

```diff
@@ -291,4 +293,31 @@ describe("ErrorUtils", () => {
 			assert.strictEqual((error as any).statusCode, 400);
 		});
 	});
+
+	describe("getUrlForTelemetry", () => {
+		// 0:hostName 1:path 2:expectedOutput
+		const testCases = [
+			["", undefined, undefined],
+			["/", undefined, undefined],
+			["http://some.url.com", undefined, "some.url.com"],
+			["http://some.url.com/", undefined, "some.url.com"],
+			["https://some.url.com/", "", "some.url.com"],
+			["something://some.url.com/", "", "something:"],
+			["some.url.com/path", undefined, "some.url.com"],
+			["some.url.com/", "randomPath", "some.url.com"],
+			["some.url.com/", socketIoPath, `some.url.com/${socketIoPath}`],
+			["http://some.url.com/", "repos", "some.url.com/repos"],
+			["some.url.com/", "deltas", "some.url.com/deltas"],
+			["https://some.url.com", "documents", "some.url.com/documents"],
+			["https://some.url.com/", "/documents/", "some.url.com/documents"],
+			["https://some.url.com/", "documents/morePath", "some.url.com/documents"],
```

Comment:
Would it be simple to do some request method testing? I see urls, but no request method

Reply:
I think he tested the getUrlForTelemetry method below which is the main logic Kian added in the PR.

Reply:
Feel free to resolve here :)